Documentation/ExtensionArchitecture/FilesAndLocations/Index.rst

```diff
@@ -436,8 +436,9 @@ Configuration/TCA
   ".php". Only for new tables.
 
 Configuration/TCA/Overrides
-  For extending existing tables, one file per database table, using the name of
-  the table for the file, plus ".php".
+  For extending existing tables.
+  General advice: One file per database table, using the name of the table for the file, plus ".php".
+  For more informations, see chapter :ref:`Extending the TCA Array <storing-in-extensions>`.
```

Comment:
Where did you get the label `storing-in-extensions` from? I can't find it in the 10.4 branch.


I assume you mean `storing-changes-extension` on https://docs.typo3.org/m/typo3/reference-coreapi/10.4/en-us/_sources/ExtensionArchitecture/ExtendingTca/StoringChanges/Index.rst.txt

Reply:
Did not know, where the get the label, I guessed it..
But you are right, thanks for patching and merging.